changelog.txt

```diff
@@ -1,3 +1,95 @@
+-------------------------------------------------------------------------------
+CorsixTH 0.68.0 - released xxxx 2024
+-------------------------------------------------------------------------------
+# New Features/Enhancements
+* In game movies now play for users using TH Data directly from a .ISO file
+* Custom campaign creators can now optionally use the original level advance
+  movie when progressing through the campaign
+* The Map Editor has received some love including new features, fixes, and
+  jukebox controls
+* You can now control the Jukebox from the Main Menu by going to
+  Options > Jukebox
+* External (non-TH) XMI files are now supported by the Jukebox
+* The adviser now tells you the cost of replacing your machine if you cannot
+  currently afford it
+* More cheats!
+* [Experimental] Right mouse panning can now be used instead of using the
+  middle mouse button. Enable it in the configuration file
+* [Experimental] We’re closer to fully implementing falling actions! Get a
+  sneak peek and get the chance to push people over by enabling it in the
+  configuration file/debug menu
+
+# Changes
+* The demo movie will no longer play and cause jumpscares at the main menu if
```

Comment:
```suggestion
* The demo movie will no longer play and cause jump-scares at the main menu if
```
not sure

Reply:
Annoyingly `jump scares`, `jump-scares`, and `jumpscares` are all acceptable in English, very helpful